backend/src/knowledge_table_api/config.py

```diff
@@ -29,7 +29,7 @@ class Settings(BaseSettings):
     unstructured_api_key: str | None = None
 
     model_config = SettingsConfigDict(
-        env_file=".env", env_file_encoding="utf-8"
+        env_file=".env", env_file_encoding="utf-8", extra="allow"
```

Comment:
It would be safer to avoid using `extra="allow"` to ensure that all configuration fields are explicitly defined. This can help prevent potential misconfigurations or unexpected behavior due to unrecognized fields in the environment file. Please, consider removing `extra="allow"` from the `SettingsConfigDict` to enforce strict validation of configuration fields.